__tests__/utils.test.js

```diff
@@ -102,3 +104,41 @@ describe("formatComments", () => {
     expect(formattedComments[0].created_at).toEqual(new Date(timestamp));
   });
 });
+
+describe.only('calculateVotes()', () => {
+  test('returns "+ 0" when passed in 0', () => {
+    const input = 0;
+    const output = calculateVotes(input);
+    const expected = "+ 0";
+
+    expect(output).toBe(expected);
+  })
+  test('returns "+ 1" when passed in 1', () => {
+    const input = 1;
+    const output = calculateVotes(input);
+    const expected = "+ 1";
+
+    expect(output).toBe(expected);
+  })
+  test('returns "+ 3" when passed in 3', () => {
+    const input = 3;
+    const output = calculateVotes(input);
+    const expected = "+ 3";
+
+    expect(output).toBe(expected);
+  })
+  test('returns "- 1" when passed in -1', () => {
+    const input = -1;
+    const output = calculateVotes(input);
+    const expected = "- 1"
+
+    expect(output).toBe(expected);
+  })
+  test('returns "- 3" when passed in -3', () => {
+    const input = -3;
+    const output = calculateVotes(input);
+    const expected = "- 3";
+
+    expect(output).toBe(expected);
```

Comment:
As a practice I think it's brilliant that you've tested a util function to convert the votes to a number. The only thing is if you're using the query syntax `$1` then PSQL will convert the data type for you. So this isn't entirely necessary, but equally does no harm

( don't forget to remove `.only` 🧹  )